test/fixtures/console/eval_messages.snapshot

```diff
@@ -1,81 +1,81 @@
 [eval]
-at *
+[eval]:1
 with(this){__filename}
 ^^^^
-
 SyntaxError: Strict mode code may not include a with statement
-    at *
-    at *
-    at *
-    at *
-    at *
-    at *
-    at *
-    at *
+    at new Script (node:vm:*:*)
+    at createScript (node:vm:*:*)
+    at Object.runInThisContext (node:vm:*:*)
+    at node:internal/process/execution:*:*
+    at [eval]-wrapper:*:*
+    at runScript (node:internal/process/execution:*:*)
+    at evalScript (node:internal/process/execution:*:*)
+    at node:internal/main/eval_string:*:*
```

Comment:
This stack trace is a step backward; this test will break if this call path ever changes, which isn’t something that should be breaking a test. Can we eliminate this from the snapshot, perhaps by updating the snapshot generation script if necessary?